pom.xml

```diff
@@ -239,6 +223,11 @@ THE SOFTWARE.
         <artifactId>jcifs</artifactId>
         <version>1.3.17-kohsuke-1</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.sshd</groupId>
+        <artifactId>sshd-core</artifactId>
+        <version>1.2.0</version> <!-- TODO 1.3.0 requires Java 8 -->
```

Comment:
🐛 I am VERY VERY afraid of it since there are known compatibility changes in the library, e.g.: https://issues.apache.org/jira/browse/SSHD-441 in SSHD Core 1.0. Now we use 0.14.0 coming from the SSH Module. It needs to get a number of thumbs up from the developer mailing list before going forward.

Any chance to avoid explicit specification?